README.md

```diff
@@ -2,27 +2,844 @@
 
 [![Dristhi](https://raw.githubusercontent.com/OpenSRP/opensrp-client/master/opensrp-app/res/drawable-mdpi/login_logo.png)](https://smartregister.atlassian.net/wiki/dashboard.action)
 
-To run this project, you need to do this:
-=========================================
+# Table of Contents
 
-1. Set the `ANDROID_HOME` environment variable to point to the location of your installed Android SDK 4.1.2 API level 16. For more information, look at [the documentation of maven-android-plugin](http://code.google.com/p/maven-android-plugin/wiki/GettingStarted).
+* [Introduction](#introduction)
+* [Features](#features)
+* [App Walkthrough](#app-walkthrough)
+   * [Sample Android Form](#2-sample-form)
+   * [Sample JSON Form](#sample-json-form)
+   * [Json Form Attributes](#json-form-attributes)
+* [Usage](#usage)
+* [Developer Documentation](#developer-documentation)
+   * [Pre-requisites](#pre-requisites)
+   * [Installation Devices](#installation-devices)
+   * [How to install](#how-to-install)
+   * [Customisations From Android Native JSON Form](#customisations-from-android-native-json-form)
 
-2. Start an Android Virtual Device. Normally, this means you need to run `android avd` and then start one of the devices there.
 
-Then, you can run `mvn clean install` in the main directory.
+# Introduction
 
-Tips and tricks (to be completed):
-=================================
+OpenSRP Client Native Form Module/app provides the ability to easily create Android forms using JSON forms. 
 
-* How to setup your Android SDK so that Maven finds it: http://pivotal.github.com/robolectric/maven-quick-start.html
+Thanks to this [Android Native JSON Form Library](https://github.com/vijayrawatsan/android-json-form-wizard) from which OpenSRP Client Native Form has been customised to fit OpenSRP Requirements
+
+> **NOTE**
+>
+> **JSON Form** is written using **JSON (syntax)** which can be found [here](https://www.w3schools.com/js/js_json_syntax.asp).
```

Comment:
let's not link to w3schools since that's a commercial site, maybe http://json.org/ is fine